prisma/prisma.types.ts

```diff
@@ -0,0 +1,22 @@
+import { Prisma } from '@prisma/client';
+
+const subscriber = Prisma.validator<Prisma.subscriptionsArgs>()({
+  select: { subscriberId: true, subscriberType: true },
+});
+
+export type Subscriber = Prisma.subscriptionsGetPayload<typeof subscriber>;
+
+const subscription = Prisma.validator<Prisma.subscriptionsArgs>()({
+  select: {
+    id: true,
+    subscriberId: true,
+    subscriberType: true,
+    topic: true,
+    createdAt: true,
+    updatedAt: true,
+    notifiedAt: true,
+    type: true,
+  },
+});
+
+export type Subscription = Prisma.subscriptionsGetPayload<typeof subscription>;
```

Comment:
Looks like some of the types can be imported directly from prisma client?
![Skjermbilde 2022-12-23 kl  22 16 49](https://user-images.githubusercontent.com/1177034/209406288-f74421d0-e802-4dd8-ab19-9383bb390a2a.png)